### ISSUE TITLE
ubuntu-fips image built with kairos framework v2.7.18-fips gets interrupted with login prompt at boot

### ISSUE THREAD

**Issue by santhoshdaivajna** [kairos-io/kairos#2375]
**Kairos version:**
v3.0.1
kairos framework image v2.7.18

**CPU architecture, OS, and Version:**
amd64, ubuntu 20.04

**Describe the bug**
I built a _ubuntu-fips_ base image with  minimal change to the existing dockerfile [here](https://github.com/kairos-io/kairos/blob/master/examples/byoi/ubuntu-fips/Dockerfile) and then created an installer with canvos.  The installer boots, gets to a login prompt. Other tty's do not have any messages. The node did not get an IP assigned from dhcp .  With v2.4.3 kairos framework the login prompt does not show up.

**To Reproduce**
update the kairos framework version to v2.7.18-fips in the ubuntu-fips/Dockerfile and update osbuilder to latest.
And then build the installer and boot it.
<img width="1176" alt="Screenshot 2024-03-20 at 9 09 16 PM" src="https://github.com/kairos-io/kairos/assets/504862/ef3d890e-7c50-4bec-b2fa-792feef1d227">

**Expected behavior**
The installer boots normally and does not prompt for user login.

**Logs**

**Additional context**
Installer at boot
<img width="715" alt="Screenshot 2024-03-20 at 9 40 01 PM" src="https://github.com/kairos-io/kairos/assets/504862/83d70426-2319-4cf3-894e-7fda3b0af139">

**Comment by Itxaka** [comment]
Immucore was broken under ubuntu 20.04 and similar due to the GLIBC version.

When we started packing all of our binaries with upx, we moved the toolchain-go luet package to be based upon bookworm instead of bullseye. That caused a bump of the GLIBC used which resulted in immucore not being able to run anymore as it could not find a compatible version.

This should have fix it by dropping the base os in which we build fips immucore to bullseye: https://github.com/kairos-io/packages/pull/761

That patch is now on framework master

**Comment by Itxaka** [comment]
@santhoshdaivajna can you try setting the framework version to `quay.io/kairos/framework:main-fips` to confirm that its fixed?

**Comment by Dr-N00B** [comment]
@Itxaka , after building ubuntu fips with `quay.io/kairos/framework:main-fips`. Getting, glibc version  2.32 not found.

I can see the libc version present in the image is 
```
ls -lrt ./usr/lib/x86_64-linux-gnu/libc.so*
lrwxrwxrwx 1 root root 12 Nov 22 13:32 ./usr/lib/x86_64-linux-gnu/libc.so.6 -> libc-2.31.so
```

![image](https://github.com/kairos-io/kairos/assets/7732089/9ee3ed7b-dd3a-49da-97a9-e5e4b54a38f9)

**Comment by Dr-N00B** [comment]
@Itxaka , tried building image with `quay.io/kairos/framework:main-fips`, and still getting GLIBC_2_32 not found.

The glibc version is still 2.31 in the image
```
ls -lrt ./usr/lib/x86_64-linux-gnu/libc.so*
lrwxrwxrwx 1 root root 12 Nov 22 13:32 ./usr/lib/x86_64-linux-gnu/libc.so.6 -> libc-2.31.so
```

![image](https://github.com/kairos-io/kairos/assets/7732089/faeb3deb-e080-44ad-a506-adc9837ded66)

**Comment by Itxaka** [comment]
@Dr-N00B yes, that is expected as we only bumped the immucore binary. I guess immucore now runs as expected in your test with the master fips framework?

If so, it confirms that the issue comes from there adn we will rebuild the agent to also be build with the same base OS with an old GLIBC!

**Comment by Dr-N00B** [comment]
@Itxaka , please share the build when it is available

**Comment by Itxaka** [comment]
Agent 2.8.8 was built with an older GLIBC and should be included in the [v2.7.22](https://github.com/kairos-io/kairos-framework/releases/tag/v2.7.22) framework image. @Dr-N00B can you test with that version?

**Comment by jimmykarily** [comment]
@santhoshdaivajna it should be fixed in v3.0.3. I'll close this now. Feel free to re-open if you have any issues.
